pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -92,6 +93,8 @@ matrix = [
         "QubitSum",
         "DiagonalQubitUnitary",
         "BlockEncode",
+        "MidMeasureMP",
```

Comment:
I don't think this has a matrix representation. Is that a problem here?

Reply:
yep, this is a measurement process.. I added this here as a temporary fix but am fixing this in PR https://github.com/PennyLaneAI/catalyst/pull/631 before making the other one ready-to-review